NOTICE

```diff
@@ -0,0 +1,18 @@
+Peco
+Copyright (c) 2024
+
+I. Included Software
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+Licensed under the Apache License 2.0.
+
+II. Used Software
```

Comment:
Can you elaborate a bit on what "used" means here?  In particular:

* The backend is written in Typescript, and there are a couple of dependencies in the `package.json` file.  I assume these dependencies are considered as "used software" and should be listed here.  Is that correct?
* The backend components are deployed using the AWS SAM framework.  In other words, there is a `template.yaml` file that can only be parsed by the `sam` CLI tool.  But that tool is not listed anywhere as a project dependency.  Should AWS SAM be considered as used software?
* Similarly, backend lambdas can only be run using Node.js.  But node is not listed as a dependency either.  Should it be listed here?
* Some backend components rely on AWS Services (as of today, API Gateway, Cognito and Lambda).  What is the status of such *required* third-party services?

Reply:
`Used` part should list the tools/providers you are using to assemble/build your software.

1. Yes, the dependencies in `package.json` should be listed there
2. No need, you distribute only the `template.yaml` and document it's a `sam` resource. So, no need to mention SAM as it's not used or distributed.
3. No need, it's a documentation standpoint.
4. In the `NOTICE`, we just need to mention what's actually used/shipped with the software. The external services can be listed in the `README` or documentation but not needed in the `NOTICE`. However, `NOTICE` is an open file: we can also list external services in a dedicated section if we want.